util/Migrator/DbScripts/2021-04-16_00_OrganizationUser_DeleteMany.sql

```diff
@@ -0,0 +1,194 @@
+-- Create sproc to bump the revision date of a batch of users
+IF OBJECT_ID('[dbo].[User_BumpAccountRevisionDateByOrganizationUserIds]') IS NOT NULL
```

Comment:
I don't see this sproc in the list of individual ones modified or added in the rest of the PR, am I missing something?

Reply:
Yeah, I combined the migrations from this PR as well into a single file: https://github.com/bitwarden/server/pull/1287

I also included these sprocs in the sqlproj file, which is why the build is failing.

I figured I'd sneak this in there since this is merging to a feature branch anyways